src/common/modal.js

```diff
@@ -133,6 +134,89 @@ export const confirm = ({
   })
 }
 
+@propTypes({
+  children: propTypes.node.isRequired,
+  icon: propTypes.string,
+  items: propTypes.object.isRequired,
+  title: propTypes.node.isRequired
+})
+class ChooseAction extends Component {
+  _resolve = val => {
+    this.props.resolve(val)
+    instance.close()
+  }
+
+  _reject = () => {
+    this.props.reject()
+    instance.close()
+  }
+
+  render () {
+    const { Body, Footer, Header, Title } = ReactModal
+
+    const {
+      icon,
+      items,
+      title
+    } = this.props
+
+    const body = _addRef(this.props.children, 'body')
```

Comment:
You don't need it because you're not using the `body` reference in your component.